packages/firebase_ml_custom/example/android/build.gradle

```diff
@@ -1,31 +1,30 @@
 buildscript {
-    repositories {
-        google()
-        jcenter()
-    }
+  repositories {
+    google()
+    jcenter()
```

Comment:
also interesting, no mavenCentral here, but yes on jcenter :thinking: ? not tracking ecosystem upheaval there but it's less important since jcenter committed to read-only access indefinite

Reply:
Generated, though I think we'll do a mass update across the board in future ourselves